src/Kiota.Builder/CodeDOM/CodeClass.cs

```diff
@@ -154,6 +276,24 @@ public DiscriminatorInformation DiscriminatorInformation
     {
         get; set;
     }
+    private bool isDisposed;
+    public void Dispose()
```

Comment:
Having to make CodeClass Disposable (and not having code namespace, call it in cascade) feels off.
I'd feel much better if we could avoid it.
What alternative could we consider here?